backend/src/main/java/saci/domain/model/Level.java

```diff
@@ -0,0 +1,28 @@
+package saci.domain.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "levels")
+public class Level {
+    @Id
```

Comment:
These kind of database implementation details should not be here (in the domain). @vascolusitan will explain this to you.